.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: 218546966473.dkr.ecr.us-east-1.amazonaws.com/circle-ci:tap-tester-v4
```

Comment:
This is a stale image please use `stitch-tap-tester` over `tap-tester-v4`.

Reply:
updated